src/Controller/Admin/ModuleController.php

```diff
@@ -63,6 +63,38 @@ public function catalogAction()
 
         $context = $this->get('mbo.cdc.context_builder')->getViewContext();
 
+        /*********************
+         * PrestaShop Account *
+         * *******************/
+        $accountsFacade = null;
+        $accountsService = null;
+        $urlAccountsCdn = '';
+
+        try {
+            $accountsFacade = $this->get('mbo.ps_accounts.facade');
+            $accountsService = $accountsFacade->getPsAccountsService();
+            $this->ensurePsAccountIsEnabled();
+        } catch (\PrestaShop\PsAccountsInstaller\Installer\Exception\InstallerException $e) {
+            $accountsInstaller = $this->get('mbo.ps_accounts.installer');
+            $accountsInstaller->install();
+            $accountsFacade = $this->get('mbo.ps_accounts.facade');
+            $accountsService = $accountsFacade->getPsAccountsService();
+        }
+
+        if (null !== $accountsFacade && null !== $accountsService) {
+            try {
+                \Media::addJsDef([
+                    'contextPsAccounts' => $accountsFacade->getPsAccountsPresenter()
+                        ->present(),
```

Comment:
You must add $this->name like such `->present($this->name)` so that we know that you are actually mbo

Reply:
Or at least your psx name (because it is not in mbo_php)

Reply:
🤔  This method doesn't accept any argument

https://github.com/PrestaShopCorp/prestashop-accounts-installer/blob/v1.0.4/src/Installer/Presenter/InstallerPresenter.php#L40